web/admin/src/SurveyResults.js

```diff
@@ -230,6 +230,6 @@ function stringifyForCsv(obj) {
 }
 
 export const getDefaultLocale = possiblyLocalized =>
-  typeof possiblyLocalized === 'object' ? possiblyLocalized.default : possiblyLocalized
+  possiblyLocalized.default ? possiblyLocalized.default : possiblyLocalized
```

Comment:
Just in case `possiblyLocalized` is null...

```js
possiblyLocalized && possiblyLocalized.default ? possiblyLocalized.default : possiblyLocalized
```